app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
```

Comment:
for better reading, you can do something like this, but that is optional:
```
    parts = command.split()
    cp = parts[0]
    old_file = parts[1]
    new_file = parts[2]

Reply:
done